.changeset/tricky-seahorses-look.md

```diff
@@ -0,0 +1,6 @@
+---
+'firebase': minor
+'@firebase/storage': minor
+---
+
+Add `storage().useEmulator()` method allowing users to set a storage emulator host and port.
```

Comment:
How about `Add `storage().useEmulator()` method to enable the storage emulator mode, allowing users to set a storage emulator host and port` ?

Reply:
Done.